app/services/search_service.rb

```diff
@@ -96,7 +96,6 @@ def filter_contacts
   def filter_articles
     @articles = current_account.articles
                                .text_search(search_query)
-                               .reorder('updated_at DESC')
```

Comment:
There should be some defined order in which the results are returned, such as by creation date or number of views. Otherwise, the output will appear random and inconsistent.

Reply:
Or is it the closest match that comes at the top?